packages/react/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { BUTTON_GROUP } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * renders the buttons
+ *
+ * @param {object} props props object
+ * @param {Array.<{copy: string}>} props.buttons array of buttons
+ * @param {string} props.buttons[].copy Button copy
+ * @param {string} props.buttons[].link URL for the button item
+ * @param {object} props.buttons[].renderIcon Optional icon type
+ * @returns {*} button components
+ */
+const ButtonGroup = ({ buttons }) => {
+  return featureFlag(
+    BUTTON_GROUP,
+    <div
```

Comment:
I am actually not sure, I know for sure that this helps in a navigation pattern but not sure if the button group counts as one?

@annawen1 @kennylam @andysherman2121 @sannminwin any thoughts on this?

Reply:
ehhh I don't mind either way

Reply:
Here is my thinking...

1. For me, it's clearly a list of buttons/actions/options for the user to take. It means more to anyone/anything reading it when it's in context to a list instead of floating aimlessly within a random `div`.
2. We are applying a priority to the list by saying the first is primary, and from there the priority diminishes. Because of this, I feel like an ordered list is the right way to go. We can't just throw these in randomly and expect them to perform the same way.
3. It can also benefit accessibility because screen readers will announce to the user that there is a list and how many items are in this list when they reach it.

Reply:
It has been updated to use `<ol>`/`<li>'s`